### ISSUE TITLE
Trident can be kept

### ISSUE THREAD

**Issue by fmohican** [garbagemule/MobArena#568]
**Short description**
https://youtu.be/2bmJfmCa9Ao

**Reproduction steps**
Just follow the video https://youtu.be/2bmJfmCa9Ao

**Details**

* **MobArena version**: last release
* **Server version**: Papper Last release
* **Stacktrace** (if applicable): -

**Additional info**

**Comment by Nesseley** [comment]
Ok, so for those who don't want to spend 1.48 watching the video:
1. Join with a class that has a trident with the Loyality enchantment. (Makes the trident fly back to the player who threw the trident)
2. Start arena and trow trident.
3. /ma leave before the trident gets back to you.
4. You now have the trident outside mobarena, as it returned to you because of the enchantment.

**Comment by garbagemule** [comment]
@fmohican for future reference, please take the issue template seriously and provide the information requested ("latest" is not a valid version, because it changes over time). It prevents me from going into grumpy old man mode where I end up closing the issue due to lack of information. If you can't put effort into reporting the problem or suggestion you have, why should I?

**Comment by fmohican** [comment]
well i'm not good at english, that is why i record and just record and open issues with the bug.

Let me bring "numbers"

Mob arena: version=0.104 (extracted from MobArena.jar/META-INF/maven/com.garbagemule/pom.properties)
Paper: git-Paper-176 (from console /version)

**Comment by garbagemule** [comment]
Thanks! 

You can get the current version of any plugin you have with `/version <plugin name>`, so `/version MobArena` should also do the trick :)

The Minecraft version is usually the most important part of the server version, since that's what we need to know to spin up a local test server for reproduction. This seems like an "any version that supports tridents" case, so it's not as important here.

**Comment by fmohican** [comment]
You mean the client version?
1.14.4

**Comment by garbagemule** [comment]
No, I did mean the server version. What you mentioned earlier is a _build number_. It doesn't carry any information about the Minecraft server version that the build is based on, so we technically can't know which Minecraft server version you're running just based on that. We have to assume that you're running [this](https://papermc.io/ci/job/Paper-1.14/176/) and not [this](https://papermc.io/ci/job/Paper-1.13/176/), but they are both build number 176.

It's all about removing ambiguity. Someone once came on IRC and said they were running the "latest MobArena build", and it turns out their build was 3 years old, and there had been a flurry of updates since :P

Anyway, we finally have the necessary information to reproduce the issue in a similar environment as yours. That's great! What's not so great is that this seems like a really, really annoying problem to solve. We may have to track projectiles and remove them when the player dies or leaves the arena - assuming it's that easy to just cancel out the boomerang effect.

**Comment by fmohican** [comment]
Thanks for make crystal-clear, yes i'm on 1.14.4 - 176
